jenkins/pr-multi-build/Jenkinsfile

```diff
@@ -0,0 +1,42 @@
+pipeline {
+    agent { label 'docker' }
+    parameters {
+        string(name: "NUM_RUNS", defaultValue: "3",
+            description: "The (integer) number of times to run the automation tests.")
+    }
+
+    stages {
+        stage('Run tests') {
+            failFast true
```

Comment:
I don't want to do it now because I want the current multi-run to finish, but I'm thinking we want to remove this line, which terminates all parallel test runs if one fails. This will allow us to see the result of all test runs, and prevent things like transient fiab errors from painting a poor picture.